MANIFEST.in

```diff
@@ -18,4 +18,5 @@ include pytest.ini
 include tox.ini
 include setuptools/tests/config/setupcfg_examples.txt
 include setuptools/config/*.schema.json
+include .latest .stable
```

Comment:
I'm slightly disinclined to use `.`-prefixed filenames. They're platform sensitive (visible by default on Windows) and IMO are just a crutch. Moreover, I'd like to [move ancillary concerns out of the root](https://github.com/coherent-oss/system/blob/main/essential-layout.md#tracking-ancillary-details).

What do you think instead about `(meta)/latest.txt` and `(meta)/stable.txt`?

Reply:
I have updated the PR to reflect that suggestion.